lib/chef/provider/windows_task.rb

```diff
@@ -229,6 +230,27 @@ def action_disable
 
         private
 
+        def task_exists?
+          task = TaskScheduler.new(new_resource.task_name, nil, "\\", false)
+          if new_resource.task_name.include?("\\")
+            *path, task_name = new_resource.task_name.split("\\")
+            path = path.join("\\")
+            begin
+              task = TaskScheduler.new(new_resource.task_name, nil, path, false)
+            rescue Exception => e
+              if e.message == "folder '#{path}' not found"
+                logger.error "Folder '#{path} not found calling create."
```

Comment:
`task_exists?` itself doesn't seem to create anything, so I'm not sure what 'calling create' implies here. Can you improve this message? I can't suggest something at the moment as I'm now sure what the behavior is yet.